src/admin/resolvers/mutations/ApprovedItem.integration.ts

```diff
@@ -860,6 +863,172 @@ describe('mutations: ApprovedItem', () => {
       );
     });
   });
+
+  describe('importApprovedCuratedCorpusItem mutation', () => {
+    const input: ImportApprovedCuratedCorpusItemInput = {
+      url: 'https://test.com/docker',
+      title: 'Find Out How I Cured My Docker In 2 Days',
+      excerpt: 'A short summary of what this story is about',
+      status: CuratedStatus.CORPUS,
```

Comment:
not a blocker: just calling out that we will be setting `RECOMMENDATION` as default for all our backfill mutations